src/Adapter/CartRule/AbstractCartRuleHandler.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
```

Comment:
Please use the new license header

Reply:
Could you elaborate? As far as I can see all files, even new ones have exactly this license header.